src/main/java/com/example/springfirsthw/repositories/BookRepository.java

```diff
@@ -2,13 +2,16 @@
 
 import com.example.springfirsthw.domain.Book;
 import java.util.List;
-import org.springframework.stereotype.Repository;
+import java.util.Optional;
 
-@Repository
 public interface BookRepository {
 
     Book save(Book book);
 
     List<Book> findAll();
 
+    Optional<Book> findById(Long id);
```

Comment:
findById() or getById()? Let's think about it

Reply:
By convention `findById` as it returns Optional